sdk/lusid/models/weighted_instrument.py

```diff
@@ -44,24 +45,11 @@ class WeightedInstrument(object):
         'instrument': 'instrument'
     }
 
-    required_map = {
-        'quantity': 'optional',
-        'holding_identifier': 'optional',
-        'instrument': 'optional'
-    }
-
-    def __init__(self, quantity=None, holding_identifier=None, instrument=None):  # noqa: E501
-        """
-        WeightedInstrument - a model defined in OpenAPI
-
-        :param quantity:  The quantity of the instrument that is owned.
-        :type quantity: float
-        :param holding_identifier:  Identifier for the instrument.  For a single, unique trade or transaction this can be thought of as equivalent to the transaction identifier, or  a composite of the sub-holding keys for a regular sub-holding. When there are multiple transactions sharing the same underlying instrument  such as purchase of shares on multiple dates where tax implications are different this would not be the case.    In an inlined aggregation request if this is wanted to identify a line item, it can be specified in the set of aggregation keys given on the aggregation  request that accompanies the set of weighted instruments.
-        :type holding_identifier: str
-        :param instrument: 
-        :type instrument: lusid.LusidInstrument
-
-        """  # noqa: E501
+    def __init__(self, quantity=None, holding_identifier=None, instrument=None, local_vars_configuration=None):  # noqa: E501
```

Comment:
What is this new `local_vars_configuration`?

Reply:
Generated again with our old model template and this is back to how it was before

Reply:
It's okay that this is new, we don't necessarily need to use the old model template here. I believe this is an added feature which allows us to skip any client side validation on the models before making the API request, if that is indeed what it is, we should include it!

Reply:
This is fine to stay with the new template.